docs/functions.rst

```diff
@@ -874,19 +874,18 @@ Example:
       outputArtifacts:
         backupStats:
           keyValue:
-            mode: "{{ .Phases.backupDataStatsFromObjectStore.Output.BackupDataStatsOutputMode }}"
-            fileCount "{{ .Phases.backupDataStatsFromObjectStore.Output.BackupDataStatsOutputFileCount }}"
-            size: "{{ .Phases.backupDataStatsFromObjectStore.Output.BackupDataStatsOutputSize }}"
+            mode: "{{ .Phases.BackupDataStatsFromObjectStore.Output.Mode }}"
```

Comment:
The output available for this function is defined here right: https://github.com/kanisterio/kanister/blob/master/pkg/function/backup_data.go#L119 ?

Reply:
Yes. Sorry, I was confused. I will undo this